packages/kbn-securitysolution-list-api/src/api/index.ts

```diff
@@ -334,36 +333,20 @@ const fetchExceptionListsItemsByListIds = async ({
   http,
   listIds,
   namespaceTypes,
-  filterOptions,
+  filter,
   pagination,
+  search,
   signal,
 }: ApiCallByListIdProps): Promise<FoundExceptionListItemSchema> => {
-  const filters: string = filterOptions
```

Comment:
This was some really funky custom code that would only take certain search params like tags and name and pass through the appropriate filter. The change to use `search` aligns with other search in our app and opens up search options beyond `tags` and `name`.